### PR TITLE
IOS Demo Setup

### PR DESCRIPTION
**Pull request by CalebKL**
This PR sets up the IOS demo for `Artmaker` and it works as expected. The demo package to run the android app has also been renamed to `androidApp`. 
Since Artmaker module has been fully been migrated to shared and we are still working on the Shared, there's no need to keep the module and so, I have deleted the module.

### CODE REVIEW

File: iosApp/iosApp/ContentView.swift [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        MainViewControllerKt.mainViewController()
+    }
+
+    func updateUIViewController(_ uiViewController: UIViewController, context: Context) {}
+}
+
+struct ContentView: View {
+    var body: some View {
+        ComposeView()
+                .ignoresSafeArea(.keyboard)
```
Comment (Fbada006) [inline iosApp/iosApp/ContentView.swift:17]: What does this mean?

File: shared/src/iosMain/kotlin/com/fbada006/shared/data/CreateDataStore.ios.kt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+package com.fbada006.shared.data
+
+import androidx.datastore.core.DataStore
+import androidx.datastore.preferences.core.Preferences
+import kotlinx.cinterop.ExperimentalForeignApi
+import platform.Foundation.NSDocumentDirectory
+import platform.Foundation.NSFileManager
+import platform.Foundation.NSUserDomainMask
+
+fun createDataStore(): DataStore<Preferences> {
```
Comment (Fbada006) [inline shared/src/iosMain/kotlin/com/fbada006/shared/data/CreateDataStore.ios.kt:12]: Nice!

**Review by Fbada006 — APPROVED** [review]
Nice! Works smoothly!

File: iosApp/iosApp/ContentView.swift (hunk 1 above)
Comment (CalebKL) [reply iosApp/iosApp/ContentView.swift:17]: I don't think is necessary. I have read more about it and I think, we can remove it. 
What it means is, when there is a trigger to a keyboard in a multi-platform project, Compose handles the keyboard positioning and resizing the content when the keyboard is shown or hidden. So there's no need to trigger any repositioning from IOS. 

I will go ahead and remove it and request a review again before meging

**Pull request merged**